core/capabilities/fakes/http_action.go

```diff
@@ -127,16 +125,6 @@ func (fh *DirectHTTPAction) SendRequest(ctx context.Context, metadata commonCap.
 	return response, nil
 }
 
-func (fh *DirectHTTPAction) Start(ctx context.Context) error {
```

Comment:
How do we start caps now without this?

Reply:
This was actually a bug - overriding Start() and Close() from the underlying Service object. @justinkaseman 's test was failing. Start() and Close() are still there and if we want to customize they need to be named differently (e.g. lowercase)